pom.xml

```diff
@@ -22,6 +21,21 @@
     <dependencies>
         <!-- needed by egads -->
         <dependency>
+          <groupId>com.fasterxml.jackson.core</groupId>
+          <artifactId>jackson-databind</artifactId>
+          <version>2.4.4</version>
+        </dependency>
+        <dependency>
+          <groupId>org.apache.logging.log4j</groupId>
+          <artifactId>log4j-api</artifactId>
```

Comment:
Not necessary; the log4j-api artifact is a dependency of the log4j-core artifact.